docs/dev-running.md

```diff
@@ -1,25 +1,8 @@
-## Running Fedimint for dev testing
-
-Fedimint consists of three kinds of executables:
-
-* **Federation nodes** - servers who form the mint by running a consensus protocol
-* **Lightning gateways** - allows users send and receive over Lightning by bridging between the mint and an LN node
-* **User clients** - handles user communication with the mint and the gateway
-
```

Comment:
This is out-of-date and not really necessary to describe initially. Let's get them to a working tmuxinator environment and then they can explore the services themselves.